src/lib/mina_base/account.ml

```diff
@@ -696,6 +696,37 @@ let create_time_locked public_key balance ~initial_minimum_balance ~cliff_time =
     ~vesting_period:Global_slot.(succ zero)
     ~vesting_increment:initial_minimum_balance
 
+let initial_minimum_balance Poly.{ timing; _ } =
+  match timing with
+  | Timing.Untimed ->
+      None
+  | Timed t ->
```

Comment:
nit: I think you can use `Option.map` for these functions, a little more concise, instead of pattern-matching

Reply:
I'd love to do that. Unfortunately, `Option.map` won't take `Account.Timing.t` as input, only an `option`…

Reply:
Sorry for the noise.